client/src/global/components/sign-in-up/form-inputs/AvatarDropzone/AvatarDropzone.js

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import Form from "global/components/form";
+import Dropzone from "react-dropzone";
+import IconComposer from "global/components/utility/IconComposer";
+import Avatar from "global/components/avatar";
+import { useTranslation, Trans } from "react-i18next";
+import { useFormContext } from "react-hook-form";
+import { useUID } from "react-uid";
+
+export default function AvatarDropzone({ errors }) {
+  const { t } = useTranslation();
+  const uid = useUID();
+
+  const { watch, setValue } = useFormContext();
+
+  const showAvatar = !!watch("avatar");
+  const removeAvatar = () => {
+    setValue("avatar", null, { shouldDirty: true, shouldValidate: false });
+  };
+  const handleDrop = file => {
+    setValue("avatar", file[0], { shouldDirty: true, shouldValidate: false });
+  };
```

Comment:
This is looking good! You could set the avatar's values here to what you're transforming them into [on submit](https://github.com/ManifoldScholar/manifold/pull/3430/files#diff-4596c2e59267d1d0e39a5c495c48b17ba021fc2774d711199f48ee0bc7526d4aR73):
```
setValue("avatar", {
   file: file[0],
   content_type: file[0].type,
   filename: file[0].name
})
```
But since you still need to get the image data from the avatar, and you don't need to read the file data until submit, I don't think this change would make much of a difference.

For removing the avatar, you might be able to set that value here. I think that's what we're doing on NGLP - setting the remove value inside the image component.
```
const removeAvatar = () => {
   setValue("removeAvatar", true);
   ...
}
const handleDrop = () => {
   setValue("removeAvatar", false);
   ...
}
```
If this component is shared, you could pass in a "removeFieldName" to this component.

All in all, my suggestions are minor improvements. This solution looks good to me as is 👍